src/gedi_subset/subset.py

```diff
@@ -41,6 +41,8 @@ class CMRHost(str, Enum):
     nasa = "cmr.earthdata.nasa.gov"
 
 
+logicals = {"L4A": "10.3334/ORNLDAAC/2056", "L2A": "10.5067/GEDI/GEDI02_A.002"}
```

Comment:
nit: slightly better name suggestion:
```suggestion
logical_dois = {"L4A": "10.3334/ORNLDAAC/2056", "L2A": "10.5067/GEDI/GEDI02_A.002"}
```